cabal.project

```diff
@@ -1,5 +1,10 @@
 packages:   streamly.cabal
-          , core/streamly-core.cabal
           , test/streamly-tests.cabal
           , benchmark/streamly-benchmarks.cabal
           , docs/streamly-docs.cabal
+
+source-repository-package
+  type: git
+  location: https://github.com/composewell/streamly
+  tag: master
+  subdir: core
```

Comment:
Undo the changes in this file.

Use some other cabal.project.* for the failing SDIST build.